packages/forma-36-website/gatsby-config.js

```diff
@@ -116,23 +116,27 @@ module.exports = {
         ],
       },
       {
-        name: 'Components',
+        name: 'Layout Components',
```

Comment:
@suevalov maybe @m10l would have some insights here, we had a discussion around the structure of the navigation recently. There was the argument here, that we should keep the main structure like it is so:
**Foundation**
**Guidelines**
**Components**
and just nest the additional section inside the **Components** - the same as we do with `Form elements`. 
WDYT?

Reply:
I can change it back but later I'd like to have a separate discussion about the navigation.

Reply:
@suevalov ok, I'm up for it, of course. Maybe we should have a brother discussion about documentation and the website.